modules/transformer-kinesis/src/main/scala/com/snowplowanalytics/snowplow/rdbloader/transformer/kinesis/parquet/ParquetSink.scala

```diff
@@ -0,0 +1,86 @@
+package com.snowplowanalytics.snowplow.rdbloader.transformer.kinesis.parquet
+
+import cats.Functor
+import cats.data.EitherT
+import cats.effect.{Blocker, Concurrent, ContextShift, Timer}
+import cats.implicits._
+import com.github.mjakubowski84.parquet4s.RowParquetRecord
+import com.github.mjakubowski84.parquet4s.parquet.viaParquet
+import com.snowplowanalytics.snowplow.badrows.FailureDetails
+import com.snowplowanalytics.snowplow.rdbloader.common.LoaderMessage.TypesInfo.WideRow
+import com.snowplowanalytics.snowplow.rdbloader.common.transformation.Transformed
+import com.snowplowanalytics.snowplow.rdbloader.common.transformation.Transformed.Data.ParquetData.FieldWithValue
+import com.snowplowanalytics.snowplow.rdbloader.common.transformation.parquet.fields.AllFields
+import com.snowplowanalytics.snowplow.rdbloader.common.transformation.parquet.{AtomicFieldsProvider, NonAtomicFieldsProvider}
+import com.snowplowanalytics.snowplow.rdbloader.transformer.kinesis.parquet.Codecs._
+import com.snowplowanalytics.snowplow.rdbloader.transformer.kinesis.sinks.{SinkPath, TransformedDataOps, Window}
+import com.snowplowanalytics.snowplow.rdbloader.transformer.kinesis.{Resources, State}
+import fs2.{Pipe, Stream}
+import org.apache.parquet.schema.MessageType
+
+import java.net.URI
+
+object ParquetSink {
+
+  def parquetSink[F[_] : Concurrent : ContextShift : Timer](resources: Resources[F],
+                                                            uri: URI)
+                                                           (window: Window)
+                                                           (path: SinkPath): Pipe[F, Transformed.Data, Unit] = {
+    transformedData =>
+
+      val targetPath = uri.toString + window.getDir + "/" + path.value
```

Comment:
I haven't followed through the code, but how sure are we at this point that `uri.toString` ends with a `/`?

Reply:
I think we are not, changing to `Path.of...` which handle it.